target.go

```diff
@@ -172,12 +172,6 @@ func createSymlink(t Target, dst string, name string, linkTarget string, cfg *Co
 	// Check if link target is absolute path
 	if filepath.IsAbs(linkTarget) {
 
-		// continue on error?
-		if cfg.ContinueOnError() {
-			cfg.Logger().Info("skip link target with absolute path", "link target", linkTarget)
-			return nil
-		}
-
```

Comment:
It seems reasonable to hard-fail here, just wondering what drove this change in this PR? Also could this break any existing users?

Reply:
Ha, great observation!

So, this branch would be reached by a symlink target that starts with an absolut path. Here is the condition before that removed check:

https://github.com/hashicorp/go-extract/blob/3c7c310952128d3d1dfbc5f547d8f876068569b1/target.go#L172-L173

The removed lines would handle the situation if `cfg.continueOnError = true`, so a `nil` would be returned. Looking into the place where the function is called:

https://github.com/hashicorp/go-extract/blob/3c7c310952128d3d1dfbc5f547d8f876068569b1/extractor.go#L408-L424

The error would never end up in the [`handleError`](https://github.com/hashicorp/go-extract/blob/3c7c310952128d3d1dfbc5f547d8f876068569b1/extractor.go#L236) function, no telemetry collected and the link would be added to the list of extracted entries here:

https://github.com/hashicorp/go-extract/blob/3c7c310952128d3d1dfbc5f547d8f876068569b1/extractor.go#L419-L421

even if the link was never created. 

tl;dr: That was a bug :-D

Reply:
Nice, thanks for the explanation, this makes sense!